qutebrowser/browser/signalfilter.py

```diff
@@ -56,9 +56,10 @@ def create(self, signal, tab):
         Return:
             A partial function calling _filter_signals with a signal.
         """
-        return functools.partial(self._filter_signals, signal, tab)
+        log_signal = debug.signal_name(signal) not in self.BLACKLIST
```

Comment:
I assume this is a performance optimization to only check this once when creating the signal filter? While you're at it, maybe make `BLACKLIST` a set instead?